src/util/Constants.js

```diff
@@ -571,17 +580,25 @@ exports.APIErrors = {
   UNKNOWN_USER: 10013,
   UNKNOWN_EMOJI: 10014,
   UNKNOWN_WEBHOOK: 10015,
+  UNKNOWN_BAN: 10026,
   BOT_PROHIBITED_ENDPOINT: 20001,
   BOT_ONLY_ENDPOINT: 20002,
+  CHANNEL_HIT_RATELIMIT: 20028,
```

Comment:
Is that name from somewhere else?
Otherwise I'd like to include `write` there. 😅 

```suggestion
  CHANNEL_HIT_WRITE_RATELIMIT: 20028,
```